src/static-pages/index.tsx

```diff
@@ -0,0 +1,298 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2013-2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link     https://github.com/serlo/serlo.org-cloudflare-worker for the canonical source repository
+ */
+import {
+  sanitizeHtml,
+  markdownToHtml,
+  LanguageCode,
+  isLanguageCode,
+  PreactResponse,
+  JsonResponse,
+  NotFoundResponse
+} from '../utils'
+import { getPathnameWithoutTrailingSlash } from '../url-utils'
+import { Template } from '../ui'
+import { h } from 'preact'
+import {
+  ALL_UNREVISED_TYPES,
+  ALL_REVSIED_TYPES,
+  UnrevisedType,
+  RevisedType,
+  Spec,
+  RevisedSpec,
+  unrevisedConfig as defaultUnrevisedConfig,
+  revisedConfig as defaultRevisedConfig,
+  UnrevisedConfig,
+  RevisedConfig,
+  Config,
+  titles
+} from './config'
+
+const defaultLanguage: LanguageCode = 'en'
+
+export {
+  RevisedType,
+  UnrevisedType,
+  Spec,
+  RevisedSpec,
+  UnrevisedConfig,
+  RevisedConfig
+} from './config'
+
+export interface Page extends Spec {
+  title: string
+  lang: LanguageCode
+}
+
+export interface RevisedPage extends Page, RevisedSpec {
+  revisedType: string
+  isCurrentRevision: boolean
+}
+
+export type WithContent<A extends Spec> = A & { content: string }
+
+export async function handleRequest(
+  request: Request,
+  unrevisedConfig = defaultUnrevisedConfig,
+  revisedConfig = defaultRevisedConfig
+): Promise<Response | null> {
+  const hostnameParts = new URL(request.url).hostname.split('.')
+
+  if (hostnameParts.length !== 3) return null
+  const [lang, secondLevelDomain, topLevelDomain] = hostnameParts
+
+  if (topLevelDomain !== 'org') return null
+  if (secondLevelDomain !== 'serlo') return null
```

Comment:
Why? Cloudflare Worker already runs only on a single domain (`serlo.org` resp. `serlo-staging.dev` resp. `serlo-development.dev`)

Reply:
There is already a helper for the subdomain